client-v2/src/shared/util/card.ts

```diff
@@ -5,12 +5,15 @@ import cloneDeep from 'lodash/cloneDeep';
 
 const createCard = (
   id: string,
+  isEdition: boolean,
```

Comment:
I'd love to refactor this into an options object at some point -- is it appropriate to do it in this PR, or is there another PR pending that will change things here?

Reply:
No time, but I'll raise a tech debt card and add a comment?

Reply:
Sounds good.